cheri_compressed_cap_common.h

```diff
@@ -897,6 +900,7 @@ static inline bool _cc_N(setbounds_impl)(_cc_cap_t* cap, _cc_addr_t req_base, _c
 
 /* @return whether the operation was able to set precise bounds precise or not */
 static inline bool _cc_N(setbounds)(_cc_cap_t* cap, _cc_addr_t req_base, _cc_length_t req_top) {
+    cap->_cr_cursor = req_base;
```

Comment:
This is odd

Reply:
To match https://github.com/CTSRD-CHERI/sail-cheri-riscv/blob/41454cad0ef41ec8e367be22f721517818224fc6/src/cheri_cap_common.sail#L368, but I guess I could rewrite this to move the assignment to _impl.